subcommand/server-acl-init/command_test.go

```diff
@@ -721,318 +638,100 @@ func TestRun_BindingRuleUpdates(t *testing.T) {
 	}
 }
 
-// Test that if the server pods aren't available at first that bootstrap
+// Test that if the servers aren't available at first that bootstrap
 // still succeeds.
-func TestRun_DelayedServerPods(t *testing.T) {
+func TestRun_DelayedServers(t *testing.T) {
 	t.Parallel()
 	require := require.New(t)
 	k8s := fake.NewSimpleClientset()
 
-	type APICall struct {
-		Method string
-		Path   string
-	}
-	var consulAPICalls []APICall
-	consulServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		// Record all the API calls made.
-		consulAPICalls = append(consulAPICalls, APICall{
-			Method: r.Method,
-			Path:   r.URL.Path,
-		})
-
-		switch r.URL.Path {
-		case "/v1/agent/self":
-			fmt.Fprintln(w, `{"Config": {"Datacenter": "dc1"}}`)
-		default:
-			// Send an empty JSON response with code 200 to all calls.
-			fmt.Fprintln(w, "{}")
-		}
-	}))
-	defer consulServer.Close()
-	serverURL, err := url.Parse(consulServer.URL)
-	require.NoError(err)
-	port, err := strconv.Atoi(serverURL.Port())
-	require.NoError(err)
+	randomPorts := freeport.MustTake(6)
 
 	ui := cli.NewMockUi()
 	cmd := Command{
 		UI:        ui,
 		clientset: k8s,
 	}
-	cmd.init()
 
 	// Start the command before the Pod exist.
 	// Run in a goroutine so we can create the Pods asynchronously
```

Comment:
```suggestion
	// Run in a goroutine so we can start the server asynchronously
```

Reply:
still need this change

Reply:
🤦‍♀️ updated, and also found a few more no longer relevant references to Pods. Ty!